tests/test_saturation.py

```diff
@@ -8,6 +8,11 @@
 
 from stcal.saturation.saturation import flag_saturated_pixels
 
+# dictionary with required DQ flags
+DQFLAGS = {'DO_NOT_USE': 1, 'SATURATED': 2, 'AD_FLOOR': 64,
```

Comment:
Might be safer to load dqflags.py and use the mnemonics from there. Hardcoding the values here could lead to problems if the dqflags definitions ever get changed in the future.

Reply:
just looked through dqflags.py, and im not sure i understand the purpose of these functions

wouldn't i still have to hard code a dictionary? i'm looking at the docstrings and it seems like i'd still have to define the exact dictionary above. the functions being tested also require a dictionary of dqflags to be passed in

Reply:
They are coded in 
```
from jwst.datamodels.dqflags import pixel
```

Reply:
Or in this case you might need to import the "group" definitions, as opposed to "pixel". But they might be copies of one another for the flags of interest here.

Reply:
im pretty sure they are the same for pixel and group. also, jwst is not a dependency of stcal right so i shouldn't import dqflags from there?

Reply:
all the tests in STCAL have a hard coded dictionary of dq flags at the top of the file, and all of them happen to use JWST's dqflags. should we do this a different way for all of the tests in another PR and just merge this as is?

Reply:
Right. Sorry, I confused you. The dqflags are passed from jwst (and romancal) in the top level calling routine.
https://github.com/spacetelescope/stcal/blob/main/src/stcal/jump/jump.py#L17